<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsComputingWizard</class>
 <widget class="QWizard" name="StatisticsComputingWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics Computing</string>
  </property>
  <widget class="QWizardPage" name="welcomePage">
   <property name="title">
    <string>Welcome</string>
   </property>
   <property name="subTitle">
    <string>Ready for computation ?</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hello there !</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to &lt;span style=&quot; font-weight:700;&quot;&gt;NIC Stats&lt;/span&gt;, a software designed to compute National Identity Cards reception statistics at &lt;span style=&quot; font-weight:700;&quot;&gt;Regional Identification Center of Maroua&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>144</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;* This software had been written by &lt;span style=&quot; font-weight:700;&quot;&gt;Amadou Wanie Benjamain&lt;/span&gt; to ease NICs dispatch statistics computations and reduce errors that may occurs during the process.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="StatisticsSetupPage" name="setupPage">
   <property name="title">
    <string>Setup</string>
   </property>
   <property name="subTitle">
    <string>Let's prepare things before processing.</string>
   </property>
  </widget>
  <widget class="StatisticsProcessingPage" name="processingPage">
   <property name="title">
    <string>Processing...</string>
   </property>
   <property name="subTitle">
    <string>We're computing...</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="completedPage">
   <property name="title">
    <string>Completed !</string>
   </property>
   <property name="subTitle">
    <string>Computing completed, look at your output directory.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Congrats, it's completed !</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>We have computed based on your data, you can get your files on the output directory.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outputDirLabel">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="openOutputDirOnCloseInput">
      <property name="text">
       <string>Open output directory on close</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatisticsSetupPage</class>
   <extends>QWizardPage</extends>
   <header location="global">stats-wizard/statisticssetuppage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatisticsProcessingPage</class>
   <extends>QWizardPage</extends>
   <header location="global">stats-wizard/statisticsprocessingpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
